ui/shared/DropdownMenuItems.jsx

```diff
@@ -30,20 +30,17 @@ const DropdownMenuItems = ({
 }) => (
   <DropdownMenu className="overflow-auto dropdown-menu-height">
     {/* Items pinned to top of dropdown */}
-    {pinned.length > 0 &&
-      options
-        .filter((item) => pinned.includes(item))
-        .sort((a, b) => a > b)
-        .map((item) =>
+    {pinned.length > 0 && (
+      <span>
```

Comment:
I think a `React.Fragment` would suite this better than a `span`.

Reply:
ah, okay, I've never used Fragment before (though I've seen it in the code)

I just knew it had to have one containing element, I'll make that change